nautobot/extras/views.py

```diff
@@ -344,11 +344,6 @@ class CustomFieldListView(generic.ObjectListView):
 class CustomFieldView(generic.ObjectView):
     queryset = CustomField.objects.all()
 
-    def get_changelog_url(self, instance):
-        """Return the changelog URL."""
-        route = "extras:customfield_changelog"
-        return reverse(route, kwargs={"name": getattr(instance, "name")})
-
```

Comment:
Since it's not likely something is subclassing `CustomFieldView` this should be safe to remove but I'm OK with leaving it and restoring the `# TODO: Remove...` note.